e2e-tests.sh

```diff
@@ -137,13 +137,17 @@ CLOUD_PROVIDER="gke"
 function initialize() {
   local run_tests=0
   local custom_flags=()
+  local parse_script_flags=0
   E2E_SCRIPT="$(get_canonical_path "$0")"
   local e2e_script_command=( "${E2E_SCRIPT}" "--run-tests" )
 
+  for i in "$@"; do
+    if [[ $i == "--run-tests" ]]; then parse_script_flags=1; fi
+  done
```

Comment:
nit: can drop the for loop

```suggestion
  [[ "$@" == *"--run-tests"* ]] && parse_script_flags=1
```

Reply:
I think the for loop is better. Your approach will also match something like `--run-tests-setup` which could cause errors that are hard to debug.

Reply:
good point